### PR TITLE
Updates to Bag json

### PR DESCRIPTION
**Pull request by jtweed**

### CODE REVIEW

File: docs/rfcs/002-archival_storage/README.md [hunk 1]
```diff
@@ -597,7 +548,7 @@ Response:
       "label": "AWS S3 - Infrequent Access"
     },
     "bucket": "bucketname",
-    "prefix": "born_digital/GC253_1046-a2870a2d-5111-403f-b092-45c569ef9476"
+    "path": "http://bucketname.s3-eu-west-1.amazonaws.com/born_digital/GC253_1046-a2870a2d-5111-403f-b092-45c569ef9476"
```
Comment (alicefuzier) [inline docs/rfcs/002-archival_storage/README.md]: is this supposed to be `url`?

File: docs/rfcs/002-archival_storage/README.md [hunk 2]
```diff
@@ -514,46 +492,19 @@ Response:
...
-        "type": "IdentifierType"
-      },
-      "value": "GC253/1046"
-    }
-  ],
+  "info": {
+    "type": "BagInfo",
+    "externalIdentifier": "a2870a2d-5111-403f-b092-45c569ef9476",
+    "externalDescription": "AIP title",
+    "payloadOxum": 435255.2,
```
Comment (kenoir) [inline docs/rfcs/002-archival_storage/README.md]: Be useful to know what the aspirational requirements around `payloadOxum` are? Do we want to verify this at some point (for MVP or after)?

File: docs/rfcs/002-archival_storage/README.md (hunk 1 above)
Comment (jtweed) [reply docs/rfcs/002-archival_storage/README.md]: Yes, will fix.

File: docs/rfcs/002-archival_storage/README.md (hunk 2 above)
Comment (jtweed) [reply docs/rfcs/002-archival_storage/README.md]: We should verify it before we verify checksums. It's intended as a fail fast.

**Pull request merged**